src/main/java/mate/academy/spring/service/impl/UserServiceImpl.java

```diff
@@ -27,6 +31,11 @@ public Optional<User> findByEmail(String email) {
         return userDao.findByEmail(email);
     }
 
+    public UserResponseDto getByEmail(String email) {
```

Comment:
For the future, it is better to return the user and make the mapping in the controller or facade